src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,43 @@
 import React from 'react';
+import { Movie } from '../../react-app-env';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+export const MovieCard: React.FC<Movie> = ({
+  title,
+  description,
+  imgUrl,
+  imdbUrl,
+  imdbId,
+}) => (
+  <div className="card" key={imdbId} data-cy="card">
```

Comment:
The `key` always should be in `map` function call, https://reactjs.org/docs/lists-and-keys.html#extracting-components-with-keys